devel/xapian-core/Portfile

```diff
@@ -104,15 +104,28 @@ if [string match {xapian-bindings-*} ${subport}] {
 
 # Java
 subport xapian-bindings-java {
+    PortGroup                   java 1.0
+
     revision                    0
 
     description                 Xapian bindings for Java
     long_description            ${description}
 
+    # Required java version
+    java.version                6+
+    # JDK port to install if required java not found
+    java.fallback               openjdk11
+
     configure.args-replace      --without-java --with-java
 
-    # TODO: libxapian_jni.jnilib gets "installed" into the work directory:
-    # https://trac.xapian.org/ticket/774
+    # jni destination violates the mtree layout
+    destroot.violate_mtree  yes
+
+    post-destroot {
+        file mkdir ${destroot}/Library/Java/Extensions
+        move ${destroot}/${workpath}/${worksrcdir}/java/built/libxapian_jni.jnilib \
```

Comment:
`${workpath}/${worksrcdir}` is more simply known as `${worksrcpath}`.

Reply:
right! I built step by step ;-)